packages/worker/native/src/lib.rs

```diff
@@ -442,46 +439,47 @@ fn worker_new(mut cx: FunctionContext) -> JsResult<JsUndefined> {
         target_url: Url::parse(&url).unwrap(),
         namespace: server_options
             .get(&mut cx, "namespace")?
-            .downcast_or_throw::<JsString, FunctionContext>(&mut cx)?
+            .downcast_or_throw::<JsString, _>(&mut cx)?
             .value(&mut cx),
```

Comment:
This 3 line preamble is a bit visually noisy. What do you think about replacing it with this (written off my head, not guaranteed to compile :smile: )

```rust
macro_rules! js_value {
    ($js_obj:ident, $js_cx:ident, $prop_name:expr, $js_type:ty) => {
        $js_obj
            .get(&mut $js_cx, $prop_name)?
            .downcast_or_throw::<$js_type, _>(&mut $js_cx)?
            .value(&mut $js_cx)
    }
}
```

Then you can just say `js_value!(server_options, cx, "namespace", JsString)`

Reply:
I'll leave that for a subsequent PR, good idea, thanks.